### PR TITLE
Fix issues appearing after deploment of PR #293

### PR DESCRIPTION
**Pull request by kedhammar**
Part of validation 23_31

### CODE REVIEW

**Comment by kedhammar** [comment]
There had been some reports of missing external packages, so I used the tool pipreqs on the repo to list all external dependencies and updated the requirements.txt file. Also added pipreqs to the dev requirements because it seems useful.

**Review by chuan-wang — APPROVED** [review]

**Comment by kedhammar** [comment]
- Thoroughly updated requirements based on running pipreqs in brand new environment on GHA
- Made a small custom Bash script to run in GHA. Job will run pipreqs and see whether it agrees with requirements.txt. Hopefully this will catch any packages that are changed in the codebase without being changed in requirements.txt in the future.

**Review by alneberg — APPROVED** [review]
Looks good! Didn't know about pipreqs before!

File: .github/workflows/lint-code.yml [hunk 1]
```diff
@@ -56,3 +62,74 @@ jobs:
...
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+
+      - name: Install editorconfig-checker
+        run: npm install -g editorconfig-checker
+
+      - name: editorconfig --> Lint files
+        run: editorconfig-checker $(git ls-files | grep -v 'test\|.py\|.md\|.json\|.yml\|.yaml\|.html')
```
Comment (alneberg) [inline .github/workflows/lint-code.yml:135]: This would exclude any file with `test` in it's name right? Is that intended?

**Review by alneberg — APPROVED** [review]
Formatting looks good!

File: .github/workflows/lint-code.yml (hunk 1 above)
Comment (kedhammar) [reply .github/workflows/lint-code.yml]: Yeah, now that you mention it I don't see a reason for it, so I'll exclude it in [b28b256](https://github.com/SciLifeLab/scilifelab_epps/pull/299/commits/b28b256de63be196daaac7a0e126a4433af5639c).

**Pull request merged**